tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = pep8,py36,py37,py38,py39,coverage
+envlist = pep8,py36,py37,py38,py39,coverage,docs
```

Comment:
AFAICS there's quite a lot of warnings coming from Sphinx:

![image](https://user-images.githubusercontent.com/3383210/131721273-5c1252e0-4d44-4594-a904-40a059452a30.png)

Do we care about any of these now?

Reply:
I will try to reduce the amount of them, but I got the filling that some of them are a bit... ehm overenthusiastic. I will take care of that tomorrow.

Reply:
Turn off these superfluous warnings if easily doable in sphinx, otherwise let's ignore them

Reply:
Also! shouldn't we run this `docs` tox env in the CI? so that we know about potential errors introduced to the docs by PRs

Reply:
Regarding warnings: WIP

Regarding building docs in CI: I have added a new variable BUILD_DOCS (similar to CHECK_COVERAGE) and I've added it to py39 run. We can consider changing the approach and instead of adding new variable we could extract it to config.yml and run tox -e docs. It should also work (I think).

Reply:
I've managed to get rid of the warnings (most of them were rather pointless, but with additional new line here and there it reduced to 50-55 warnings `duplicate label`. This was handled by adding `included` files into exclude patterns (see https://github.com/sphinx-doc/sphinx/issues/1668)